### PR TITLE
Main page scaffold

### PR DESCRIPTION
**Pull request by Drapegnik**
- update almost all dependencies
- introduce root `_app` component
- refactor `utils/request`
- from now every top-level component (page) receive: `user`, `lang` and `routerQuery`
- refactor `articles.fetchChunk` & `diary.getClosest` actions
- add main page scaffold (data fetching, localization, render logic):
screen shot 2019-02-21 at 04 19 37

### CODE REVIEW

**Comment by Drapegnik** [comment]
[new "static" main page](https://trello.com/c/WX8sSOKn/301-new-static-main-page)

**Comment by Drapegnik** [comment]
[refactor populate method to support redux-thunk](https://trello.com/c/dLcFiN5g/192-refactor-populate-method-to-support-redux-thunk)

**Review by UladBohdan — APPROVED** [review]
Solid💪

File: pages/_app.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+              <Component {...defaultPageProps} {...pageProps} />
+            </CoreLayout>
+          </LocaleContext.Provider>
+        </Provider>
+      </Container>
+    );
+  }
+}
+
+export default withRouter(withRedux(initStore)(Root));
```
Comment (tyndria) [inline pages/_app.js:99]: 👍

**Review by tyndria — APPROVED** [review]

**Pull request merged**